ui/admin/app/styles/app.scss

```diff
@@ -18,6 +18,13 @@
   }
 }
 
+//fix for sidebar wrappper around centered content
+[class*='sidebar-content-centered'] {
```

Comment:
Not 100% about this solution.  Probably could conditionally change this layout somewhere else.  thoughts?  Challenge is not create a layout that will take care of the `login` and `change-password` views.  Both of those views are wrapped in the `sidebar-layout`,